bin/install-theos

```diff
@@ -61,6 +61,34 @@ fi
 
 
 # The work
+ver_compare() {
```

Comment:
nit: may you add a comment to this function? e.g.
"compares two version strings in the form of `x.y.z`. returns `1` on failure or if `$1` is less than `$2` otherwise `0`"

Reply:
Also, I'm not familiar enough with bash, but if `want == have` is the return value well-defined?